src/sources/heroku_logs.rs

```diff
@@ -44,12 +44,15 @@ use crate::{
 #[derive(Clone, Debug)]
 pub struct LogplexConfig {
     /// The address to listen for connections on.
+    #[configurable(metadata(docs::examples = "0.0.0.0:80"))]
+    #[configurable(metadata(docs::examples = "localhost:8080"))]
     address: SocketAddr,
 
     /// A list of URL query parameters to include in the log event.
     ///
     /// These will override any values included in the body with conflicting names.
     #[serde(default)]
+    #[configurable(metadata(docs::examples = "application", docs::examples = "source"))]
```

Comment:
I left this as one line as it made sense to me mentally as this is an array of strings, rather than two individual options - compared to above.

Reply:
Is this decision mostly based on line length? Or something else?

Reply:
Completely at a whim, to be honest. This one "made sense" to me as the example is rendered as an array:
```
[
  "application",
  "source"
]
```

It matched the `exec`'s `command` example:
```rust
    #[configurable(metadata(docs::examples = "echo", docs::examples = "Hello world!"))]
```

Reply:
Ah, I see, the option is an array. I would have expected `docs::examples` to accept an array then like `#[configurable(metadata(docs::examples = ["application", "source"]))]`. For the `exec` source, how would I provide multiple example commands, each with multiple args?

Reply:
@tobz is more familiar, but raw strings only for now in the `examples` and between that and the existing site we can't do something like `docs::examples = "[\"application\", \"source\"])`

With these array types (site side) they'd all get rendered into the same array - so only a single example command :/